### ISSUE TITLE
predictor.explain() only in Jupyter-Notebooks

### ISSUE THREAD

**Issue by caesar-git** [amaiya/ktrain#35]
Hello and thanks for the amazing work.

I found that the predictor.explain() method only works for me in Jupyter-Notebooks and not in standard python scripts with console or an IDE as output (which should be the same). Is this by design or am i missing something? If not, please make it work or give me a hint.

Regards,
caesar

**Comment by amaiya** [comment]
Hello:  The plotting and visualizations (e..g., `TextPredictor.explain`, `TopicModel.visualize_documents`) are intended to be run in a Jupyter notebook.  That said, `explain` is powered by the [eli5](https://github.com/TeamHG-Memex/eli5) library, which does have an option to produce text-only output for console, I think.  I've never tried it, but if you need console output you can probably just call `eli5` directly with your model and the example in question.  See  [formatters](https://eli5.readthedocs.io/en/latest/autodocs/formatters.html) for `eli5`.   Thanks.
